.github/workflows/03_github_workflows_assemble.yml

```
@@ -0,0 +1,19 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Make setup and test scripts executable
+        run: chmod +x ${{ github.workspace }}/tools/*.sh ${{ github.workspace }}/tests/*.sh
+
+      - name: Run assemble_src.sh
+        run: ${{ github.workspace }}/tools/lib_assemble_src.sh
+
+      - name: Run assemble_array.sh
+        run: ${{ github.workspace }}/tools/lib_assemble_array.sh
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)